### PR TITLE
Defer to github releases page for latest RxJava version number

### PR DESCRIPTION
**Pull request by asiplas**
see https://github.com/ReactiveX/RxJava/releases/tag/v2.1.9

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
We should just point people to RxJava [releases page](https://github.com/ReactiveX/RxJava/releases)  
 and stop updating it here

**Comment by asiplas** [comment]
Seems logical as long as user realizes there are 2 running versions—currently 1.3.5 showing above 2.x on RxJava release page.

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Since RxAndroid is part of the RxJava family the communication channels are simi
 compile 'io.reactivex.rxjava2:rxandroid:2.0.1'
 // Because RxAndroid releases are few and far between, it is recommended you also
 // explicitly depend on RxJava's latest version for bug fixes and new features.
-compile 'io.reactivex.rxjava2:rxjava:2.1.8'
+compile 'io.reactivex.rxjava2:rxjava:2.x.x' // see https://github.com/ReactiveX/RxJava/releases for latest 2.x.x release
```
Comment (artem-zinnatullin) [inline README.md]: Comment goes behind right end of the screen due to codeblock formatting

Can you try to play with it to achieve better readability? Maybe add it on line above :)

**Comment by artem-zinnatullin** [comment]
Also let's update PR title then!

**Review by artem-zinnatullin — APPROVED** [review]
Not super pretty, but functional

**Pull request merged**